### PR TITLE
Update docs to match actual response [SA-15353] master

### PR DESCRIPTION
**Pull request by tomcdonnell**
See comments in corresponding PR for develop (https://github.com/alaress/schoolbox-api-docs/pull/119).

### CODE REVIEW

File: openapi/components/schemas/news/topic.yaml [hunk 1]
```diff
@@ -2,8 +2,6 @@ type: object
 description: |
   A topic which may be applicable to a news article.
 properties:
-  id:
```
Comment (armatronic) [inline openapi/components/schemas/news/topic.yaml:5]: This change means that the documented API for `/news/topics` is no longer documented correctly.

This endpoint (e.g. https://vagrant.dev.schoolbox.cloud/news/topics) returns the `id`, `name` and `sequence` fields (but not `slug` for some reason).

**Review by armatronic — COMMENTED** [review]
Other than that I think both this and the master PR are okay to go.

File: openapi/components/schemas/news/topic.yaml (hunk 1 above)
Comment (stevechan) [reply openapi/components/schemas/news/topic.yaml:5]: I guess there was no usage of the topic slug in the ui when mr matt made it.

```
    /**
     * @return array
     */
    public function getDefaultJsonMap(): array
    {
        return [
            'id',
            'name',
            'sequence',
        ];
    }
```

Comment (tomcdonnell) [reply openapi/components/schemas/news/topic.yaml]: Thanks, fixed.

Comment (armatronic) [reply openapi/components/schemas/news/topic.yaml]: Let's not try to solve too many problems at once here.

For now it's fine that `GET /news/lists/feed` returns a sequence number as well: it doesn't conflict with the documentation either, as objects are allowed to have additional properties by default (see https://json-schema.org/understanding-json-schema/reference/object.html#additional-properties).

**Pull request merged**